doozer/tests/cli/test_verify_image_grades.py

```
@@ -0,0 +1,405 @@
+        mock_mr.title = "Shipment for 4.18.51"
+        mock_mr.source_branch = "shipment-4.18.51"
+        mock_mr.source_project_id = 123
+        mock_mr.diffs.list.return_value = [MagicMock(id=1)]
+        mock_mr.diffs.get.return_value = mock_diff
+
+        mock_gl = MagicMock()
+        mock_gl.get_mr_from_url.return_value = mock_mr
+        mock_gl.get_project.return_value = MagicMock()
+        mock_gl_cls.from_url.return_value = mock_gl
+
+        components, version = fetch_shipment_components("https://gitlab.example.com/mr/1")
```

Ruff (0.16.0)

[warning] Unpacked variable `version` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)